cmd/call-endpoint/main.go

```diff
@@ -34,13 +37,24 @@ func main() {
 		return
 	}
 
-	var endpointDescription *core.EndpointDescription
+	if args.List {
+		listAPIEndpoints()
+		return
+	}
+
+	if args.Endpoint == "" {
+		log.Error("Please enter an endpoint. Use --list to view all endpoints.")
+
+		// Return to prevent further execution after log.Error().
```

Comment:
Last round there was a comment that removed this same substring (these two lines) from the case above.
Remember that style comments generally apply to the entire PR.